src/Tool/Program.cs

```diff
@@ -74,6 +74,7 @@ public static int Main(string[] args)
                         {
                             ["TELEMETRY_OPT_OUT"] = nameof(TelemetryOptOut),
                             ["USER_AGENT"] = "UserAgent",
+                            ["USER_AGENT_EXTRA"] = "UserAgentExtra",
```

Comment:
What is this change for? I don't see where it's used elsewhere in the PR

Reply:
Apologies, forgot to note that. @xinyi-joffre had noted that we allowed overriding other properties via environment variables, but that `UserAgentExtra` was inadvertently omitted.